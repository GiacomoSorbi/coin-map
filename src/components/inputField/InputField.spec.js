import React from 'react';

import InputFields from './index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const fn = jest.fn();
  const sampleRender1 = renderer
    .create(
      <InputFields
        id="pippi"
        label="test input for Pippi"
        type="number"
        onChange={fn}
        min={0}
        placeholder="Pippi number here..."
      />
    )
    .toJSON();
  expect(sampleRender1).toMatchSnapshot();
  const sampleRender2 = renderer
    .create(
      <InputFields
        id="pappa"
        label="test input for Pappa"
        type="email"
        onChange={fn}
        placeholder="Pappa email here..."
      />
    )
    .toJSON();
  expect(sampleRender2).toMatchSnapshot();
});
