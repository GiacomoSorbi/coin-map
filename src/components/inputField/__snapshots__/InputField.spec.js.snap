// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="input-field"
>
  <label
    htmlFor="pippi"
  >
    test input for Pippi
  </label>
  <input
    id="pippi"
    min={0}
    onChange={[MockFunction]}
    placeholder="Pippi number here..."
    type="number"
  />
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="input-field"
>
  <label
    htmlFor="pappa"
  >
    test input for Pappa
  </label>
  <input
    id="pappa"
    onChange={[MockFunction]}
    placeholder="Pappa email here..."
    type="email"
  />
</div>
`;
